src/dev/flang/tools/Fuzion.java

```diff
@@ -690,8 +692,16 @@ private Runnable parseArgsAceMode(String[] args)
    */
   private Runnable parseArgsForBackend(String[] args)
   {
+    var argsLeft = args;
+
     for (var a : args)
       {
+        argsLeft = Arrays.copyOfRange(argsLeft, 1, argsLeft.length);
+        if (a.equals("--"))
+          {
+            /* stop argument parsing */
+            break;
```

Comment:
do not use `break`, this causes surprising control flow. 

Instead: Make `argsLeft` an `ArrayList<String>` that is set to `null` outside the for-loop. Set it to `new ArrayList<>()` if `"--"` is encountered. Before the `if`, add another 
```
  if (argsLeft != null) 
    {
      argsLeft.add(a);
    }
  else`if (a.equals("--"))
```